examples/meanio/README.md

```diff
@@ -0,0 +1,265 @@
+
+### Step Two: Fire up the MongoDB service
+A Kubernetes 'service' is a named load balancer that proxies traffic to *one or more* containers. This is done using the *labels* metadata which we defined in the meanio-mongodb pod above.
+
+The services in a Kubernetes cluster are discoverable inside other containers via environment variables.
+
+Services find the containers to load balance based on pod labels.
+
+The pod that you created in Step One has the label `name=meanio-mongodb`. The selector field of the service determines *which pods will receive the traffic* sent to the service, and the port and containerPort information defines what port the service proxy will run at. 
+
+Use the file `examples/meanio/meanio-mongodb-service.json`:
+
+```js
+{
+  "id": "meanio-mongodb",
+  "kind": "Service",
+  "apiVersion": "v1beta1",
+  "port": 27017,
+  "containerPort": 27017,
+  "labels": {
+    "name": "meanio-mongodb"
+  },
+  "selector": {
+    "name": "meanio-mongodb"
+  }
+}
+```
+
+to create the service by running:
+
+```shell
+$ kubectl create -f examples/meanio/meanio-mongodb-service.json
+meanio-mongodb
+
+$ kubectl get services
+NAME                    LABELS                                    SELECTOR                     IP                  PORT
+meanio-mongodb          name=meanio-mongodb                                             name=meanio-mongodb          10.0.222.168        27017
+```
+
+This will cause all pods to see the MongoDB server apparently running on <ip>:27017.
+
+- Traffic will be forwarded from the service "port" (on the service node) to the  *containerPort* on the pod which (a node the service listens to). 
+
+Thus, once created, the service proxy on each minion is configured to set up a proxy on the specified port (in this case port 27017).
+
+
+make sure that the mongodb pod is running and listening by running:
+
+```shell
+
+$ kubectl log meanio-mongodb-controller-nyxxv
+
+```
+
+### Step Three: Create the meanio pod
+
+This is a nodejs server that is configured to talk to the MognoDB server. 
+Note that you will need the MongoDB server up and running before running the meanio pod.
+
+The pod is described in the file `examples/meanio/meanio-controller.json`:
+
+```js
+{
+  "id": "meanio-controller",
+  "kind": "ReplicationController",
+  "apiVersion": "v1beta1",
+  "desiredState": {
+    "replicas": 1,
+    "replicaSelector": {"name": "meanio"},
+    "podTemplate": {
+      "desiredState": {
+         "manifest": {
+           "version": "v1beta1",
+           "id": "meanio",
+           "containers": [{
+             "name": "meanio",
+             "image":"shaiweinstein/k8s:v1",
+             "cpu": 100,
```

Comment:
+@brendandburns  What are good limits for "cpu" and "memory" here? Should we even include them?